src/ethereum_spec_tools/evm_tools/utils.py

```diff
@@ -12,6 +14,39 @@
 
 W = TypeVar("W", Uint, U64, U256)
 
+EXCEPTION_MAPS = {
+    "FrontierToHomesteadAt5": {
```

Comment:
this logic can be done also on testers side. if tool is transition only it does not keep blockchain data.
so the testing tool can run it with Frontier or Homestead to build a transition chain